src/markdown-pages/about_Galasa.md

```diff
@@ -3,19 +3,41 @@ path: "/about"
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 
+
+### Technology integration
+Galasa is fully extensible, supporting integration across technologies by using an API that interacts with multiple test tools, including JMeter, Selenium, JCL and 3270 screens and more. 
+
+### Pipeline integration
+Galasa provides a REST endpoint that can be called from any CI/CD pipeline, for example, a Jenkins pipeline. Running Galasa as part of your pipeline enables fast, reliable testing at scale - Galasa can run thousands of tests in parallel. 
+
+### Enterprise integration
+Galasa is not simply about providing the ability to write a test and run it in automation.  Galasa's test catalog enables you to understand and manage your test corpus, making it easy to schedule the right tests to run at the right time. Test results and artifacts are stored in a single repository, making reporting and problem determination simple when combined with native Kibana and Grafana dashboards. 
 
 Galasa is open-source, so there’s no vendor lock-in; it can be integrated into an existing pipeline and work with third-party tooling. 
 
-Galasa is constructed with scalability in mind, to serve enterprise requirements and work across multiple technologies to enable reliable, repeatable automated testing.
+
+
+## Using Galasa in a hybrid cloud environment
+
+Using hybrid cloud, enterprises can host applications and data on the platforms that deliver the best combination of efficiency, security, and performance. However, a mix of platforms and technologies makes end-to-end integration testing complicated.
+
+Galasa simplifies testing in such an environment. The following diagram shows an example of how you can use Galasa to test a hybrid cloud application:  
+
+![Galasa in hybrid cloud](galasa-hybrid-cloud.svg)
+
+This sophisticated solution requires end-to-end integration testing of an application that runs on different platforms (z/OS and Cloud) and uses different technologies (a 3270 emulator, JCL batch job and Selenium Web Driver). 
+
+The Galasa test can be invoked from an IDE or as part of a CI/CD pipeline (a Jenkins pipeline in this scenario). The Galasa framework initializes the test environment, creates valid test data, runs the test and validates the test output. As all test results and artifacts are stored in one location, it's easy to generate reports and diagnose the cause of any failures.

```

Comment:
should we mention that the galasa ecosystem is a cloud native application exposing a sequence of micro services that are used to manage the set of running tests?

Reply:
I have added that line in the Galasa ecosystem doc and added a link to that topic from the About section.